test/src/module/command/verifyTest.c

```diff
@@ -256,9 +257,26 @@ testRun(void)
         harnessLogLevelSet(logLevelDetail);
 
         backupResult.status = backupValid;
-        TEST_ASSIGN(manifest, verifyManifestFile(&backupResult, NULL, false, infoPg, &jobErrorTotal), "verify manifest");
+        TEST_ASSIGN(manifest, verifyManifestFile(&backupResult, NULL, false, infoPg, &jobErrorTotal, errorList), "verify manifest");
         TEST_RESULT_PTR(manifest, NULL, "manifest not set - pg version mismatch");
         TEST_RESULT_UINT(backupResult.status, backupInvalid, "manifest unusable - backup invalid");
+        TEST_RESULT_UINT(varLstSize(errorList), 2, "error list size");
+
+        TEST_RESULT_STR_Z(
+            jsonFromVar(varNewVarLst(errorList)),
+            "["
+            "{"
+            "\"level\":5,"
+            "\"message\":\"unable to open missing file '" TEST_PATH "/repo/backup/db/20181119-152138F/backup.manifest.copy' for read\""
+            "},"
+            "{"
+            "\"level\":4,"
+            "\"message\":\"'20181119-152138F' may not be recoverable - PG data (id 1, version 9.6, system-id " HRN_PG_SYSTEMID_95_Z ") is not in the backup.info history, skipping\""
+            "}"
+            "]",
+            "errorList does not match"
+            );
```

Comment:
```suggestion
            "["
                "{"
                    "\"level\":5,"
                    "\"message\":\"unable to open missing file '" TEST_PATH "/repo/backup/db/20181119-152138F/backup.manifest.copy' for read\""
                "},"
                "{"
                    "\"level\":4,"
                    "\"message\":\"'20181119-152138F' may not be recoverable - PG data (id 1, version 9.6, system-id " HRN_PG_SYSTEMID_95_Z ") is not in the backup.info history, skipping\""
                "}"
            "]",
            "errorList does not match"
            );
```
Inside infoTest.c is [used](https://github.com/arenadata/pgbackrest/blob/8cdd9ce1c4ab6cca508932a41a3013374d7547ef/test/src/module/command/infoTest.c#L164-L190) this style I think it's more readable.